tests/phpunit/tests/rest-api/rest-post-meta-fields.php

```diff
@@ -891,8 +891,8 @@ public function test_set_value_multiple_custom_schema() {
 		$meta = get_post_meta( self::$post_id, 'test_custom_schema_multi', false );
 		$this->assertNotEmpty( $meta );
 		$this->assertCount( 2, $meta );
-		$this->assertContains( 2, $meta );
-		$this->assertContains( 8, $meta );
+		$this->assertContains( '2', $meta );
+		$this->assertContains( '8', $meta );
```

Comment:
1000 x yes. The values come back from WP as strings, not integers.

As per the commit message of the [patch in which I made this change](aee3b8d7facc7ac8149f2a4e3d1192b8c6798e41):

> Tests: fix tests failing due to `assertContains()` using strict checking
> 
> Since PHPUnit 8.0.2, the `assertContains()` method when checking whether a value exists in an array will do a strict type comparison of the values.
> 
> This caused a couple of tests to fail.
> 
> Using the correct data type in the test fixes that.
> 
> Refs:
> * https://github.com/sebastianbergmann/phpunit/blob/8.0.6/ChangeLog-8.0.md#802---2019-02-07
> * https://github.com/sebastianbergmann/phpunit/issues/3511
> * https://github.com/sebastianbergmann/phpunit/commit/6205f335954d00fa01992fe324a7eefbc954449e